class.jetpack-connection-banner.php

```diff
@@ -0,0 +1,435 @@
+							esc_html_e(
+								'Jetpack is a free plugin that utilizes powerful WordPress.com servers to simplify 
+									managing your site(s) with detailed visitor stats, security services, 
+									image performance upgrades, and more. Connect with WordPress.com to get started!',
+								'jetpack'
+							);
+							?>
+						</p>
+
+						<p class="jp-banner__button-container">
+							<a
+								href="<?php esc_url( Jetpack::init()->build_connect_url( true, false, 'banner-44-slide-1' ) ); ?>"
+								class="dops-button is-primary"
+								id="wpcom-connect">
+								<?php esc_html_e( 'Connect to WordPress.com', 'jetpack' ); ?>
+							</a>
+							<a
+								href="#"
+								class="dops-button next-feature"
+								title="<?php
+								esc_attr_e(
+									'Start tour to Learn about the benefits you receive when you connect Jetpack to WordPress.com',
+									'jetpack'
+								);
+								?>">
+								<?php esc_html_e( 'Start quick tour', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div> <!-- end slide 1 -->
+
+					<!-- slide 2: stats -->
+					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-two">
+						<h2><?php esc_html_e( 'Detailed stats and traffic tools to help you grow', 'jetpack' ) ?></h2>
+
+						<div class="jp-wpcom-connect__content-icon">
+							<svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="687 83 100 100" version="1.1"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" transform="translate(687.000000, 83.000000)"><circle fill="#3D596D" cx="50" cy="50" r="50"/><path d="M45.5 63L55.5 63 55.5 28 45.5 28 45.5 63ZM60.5 63L70.5 63 70.5 35.5 60.5 35.5 60.5 63ZM30.5 63L40.5 63 40.5 45.5 30.5 45.5 30.5 63ZM28 73L73 73 73 68 28 68 28 73Z" fill="#FFFFFF"/></g></svg>
+						</div>
+
+						<p>
+							<?php
+							esc_html_e(
+								'Jetpack offers detailed stats and insights on your viewers. Make informed decisions 
+									about your content and drive more traffic to your site with our related posts and 
+									enhanced distribution features.',
+								'jetpack'
+							);
+							?>
+							<a href="https://jetpack.com/features">
+								<?php esc_html_e( 'Learn More', 'jetpack' ); ?>
+							</a>
+						</p>
+
+						<p class="jp-banner__button-container">
+							<a href="<?php echo esc_url( Jetpack::init()->build_connect_url( true, false, 'banner-44-slide-2' ) ); ?>" class="dops-button is-primary" id="wpcom-connect">
+								<?php esc_html_e( 'Connect to WordPress.com', 'jetpack' ); ?>
+							</a>
+							<a href="#" class="dops-button next-feature" title="<?php esc_attr_e( 'Jetpack Tour: Next Feature', 'jetpack' ); ?>">
+								<?php esc_html_e( 'Next feature', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div> <!-- end slide 2 -->
+
+					<!-- slide 3: security -->
+					<div class="jp-wpcom-connect__slide jp-wpcom-connect__slide-three">
+						<h2><?php esc_html_e( 'Detailed stats and traffic tools to help you grow', 'jetpack' ) ?></h2>
```

Comment:
Good find D. @ebinnion Perhaps we can use "Security tools to help your peace of mind." or something like that. I haven't heard back from growth on any language changes yet though.